conf/normalized_types.json

```diff
@@ -243,5 +243,13 @@
       "srcuser",
       "dstuser"
     ]
+  },
+  "trendmicro": {
+    "instanceID": [
```

Comment:
Do you want to rename it to `hostId` to be more generic? The key name convention is following Common Event Format white-paper, and you can find [here](https://kc.mcafee.com/resources/sites/MCAFEE/content/live/CORP_KNOWLEDGEBASE/78000/KB78712/en_US/CEF_White_Paper_20100722.pdf).

Reply:
That's fine if we use it for things like cloudtrail to map `instance_id` to hostId too?

Reply:
yeah, sounds good to me.

Reply:
Actually removed the `instanceId`, as this is not something tracked by the other schemas. This information is nested within the records so not sure normalization covers these keys. `sourceAccount` is defined on others so that makes sense to keep